.dockerignore

```diff
@@ -8,7 +8,7 @@
 terraform/.terraform
 !Makefile
 !requirements.txt
-!test/goss/
```

Comment:
What's the reason for this change ?

Reply:
`test/terraform` needs to be in docker for integration tests in https://github.com/aeternity/terraform-aws-aenode-deploy/pull/1 to run

Reply:
removed as actually not used